nixos/release.nix

```diff
@@ -1,6 +1,8 @@
 with import ../lib;
 
-{ nixpkgs ? { outPath = cleanSource ./..; revCount = 130979; shortRev = "gfedcba"; }
+# `revCount` and `shortRev` are placeholders, that should be filled
+# automatically for releases by passing the `nixpkgs` argument
+{ nixpkgs ? { outPath = cleanSourceForImport ./..; revCount = 654321; shortRev = "gfedcba"; }
```

Comment:
This revcount isn't a placeholder, it is updated when new channels are cut.

Reply:
@grahamc Are you sure about that? I ran `git log -p nixos/release.nix` before writing this comment, and it was set to `56789` since May 2014 before 18.03 cut-off (and before that it was set to `5678` since Oct 2013, and before that to a mix of `1234` for `nixosSrc` and `5678` for `nixpkgs`), and it hasn't been updated when cutting out 18.09. This is the reason why I thought @vcunat's updating it when 18.03 was cut was potentially a mistake, but…?

Reply:
@grahamc I'd like to try and move this forward. Could you either confirm, infirm or say you don't have time to evaluate my statement above? :)

Reply:
I don't have time to dive deeper, however, I thought:

>  Update versionSuffix in nixos/release.nix, use git log --format=%an|wc -l to get the commit count 

was supposed to include updating this count. (https://nixos.org/nixos/manual/index.html#at-beta-release-time)

Maybe I'm wrong, and I trust you to do the digging, and I don't feel like I need to be part of making the decision if this is true.